### DISCUSSION TITLE
I can't see the mail of the user

### DISCUSSION THREAD

**Discussion by isamess** [emailjs-com/help#119] [Q&A]
I have a free account in EmailJS, it works ok, but I need to know the  user mail to reply and I don´t know how to configure it to give me that information. Can anybody help me, please?

**Comment by xr0master** [comment]
There is too little information. Try to explain step by step how you are using the service and what result you want to achieve.

**Comment by isamess** [comment]
Excuse me for the lack of information but I have already fix it. Thank You very mucho for answering so fast.
Isa
Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by Zaknight8302** [reply]
How did you resolve that

**Comment by isamess** [comment]
I'm not very sure. I tried a couple of things, but I think the problem was
that I was working in the production context and when I did the deployment,
the changes didn't appear. When I worked in a development context it
worked. But I also think that my browser wasn't working correctly as well.
(Forgive my English, please!)

El sáb, 22 jun 2024 a las 6:23, Zaknight8302 ***@***.***>)
escribió:

> How did you resolve that
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/emailjs-com/discussions/119#discussioncomment-9846014>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUVLPI4K4A7ET4S67VBKAULZIU7APAVCNFSM6AAAAABJWUXK56VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQNBWGAYTI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Isa

LinkedIn
https://www.linkedin.com/in/isabelmesina

GitHub
https://github.com/isamess/

Instagram
https://www.instagram.com/isadisenioweb/

https://www.instagram.com/elkultrundeisaarteenvitraux

Face
https://www.facebook.com/elkultrundeisa/

Site
https://artesenvitraux.com
